src/components/basics/Pre.tsx

```diff
@@ -43,6 +43,7 @@ export function Pre({ children }) {
         />
       );
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
The amount of times I'm seeing this rule disabled is a bit worrying. Should we have a follow-up to investigate further?

Reply:
Yea I agree. I do want to fix that but maybe as the next pass. The code didn't change I just updated eslint which started flagging for this.